src/arrayMethodJoin.js

```diff
@@ -4,8 +4,23 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    const array = [...this];
```

Comment:
No need to include data type in the name of a variable. We know that it's an array. 
```suggestion
    const elements = [...this];
```